service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/NoticeOfChangeContentProvider.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.RespondentSolicitor;
+import uk.gov.hmcts.reform.fpl.model.notify.noticeofchange.NoticeOfChangeRespondentSolicitorTemplate;
+import uk.gov.hmcts.reform.fpl.service.email.content.base.SharedNotifyContentProvider;
+
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+import static org.apache.commons.lang3.StringUtils.isBlank;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class NoticeOfChangeContentProvider extends SharedNotifyContentProvider {
```

Comment:
Was thinking about merging the two functions below into one as they are pretty similar but not sure if that would be worse for readability. We'd need a third parameter to decide if to add the case URL and something like 
`buildNoticeOfChangeRespondentSolicitorTemplate(caseData, solicitor, True)` seems like it would take more effort to read than `buildRespondentSolicitorAccessGrantedNotification(caseData, solicitor)` which is a bit more obvious at a glance